src/main/kotlin/uk/gov/justice/digital/hmpps/hmppsrestrictedpatientsapi/services/SubjectAccessRequestService.kt

```diff
@@ -23,9 +23,9 @@ class SubjectAccessRequestService(
         content = RestrictedPatientContent(
           prisonerNumber = it.prisonerNumber,
           supportingPrisonId = it.supportingPrisonId,
-          supportingPrisonDescription = prison?.description,
+          supportingPrisonDescription = prison?.description ?: it.supportingPrisonId,
```

Comment:
the subject access request service should only display the descriptions and not the ids / codes.  Therefore easier to default the values here rather than attempt to do it in a moustache template.  Note that they should always exist anyway as agencies can't be deleted, but better to be safe and display the data we have than to cause the subject access request to keel over in a heap...